ds-caselaw-ingester/lambda_function.py

```diff
@@ -136,6 +133,12 @@ def send_retry_message(original_message: Dict[str, Union[str, int]], sqs_client:
         raise MaximumRetriesExceededException(f'Maximum number of retries reached for {original_message["consignment-reference"]}')
 
 
+def create_error_xml_contents(tar, consignment_reference: str):
+    parser_log = tar.extractfile(f'{consignment_reference}/parser.log')
+    parser_log_contents = parser_log.read().decode('utf-8')
+    return f'<error>{parser_log_contents}</error>'
```

Comment:
Only slight concern here is what if the parser log contained anything that might confuse the XML? It should probably be escaped somehow?